src/test/java/xbot/common/injection/MockTimerTest.java

```diff
@@ -19,7 +19,7 @@ public class MockTimerTest extends BaseWPITest {
 
     @Before
     public void setup() {
-        timer = injector.getInstance(MockTimer.class);
+        timer = new MockTimer();
```

Comment:
❔ curious about the swap here to note use the injector anymore, in the future if MockTimer had injectable dependencies the previous version would have 'just worked'

Reply:
I fixed the injector backwards compat later, I'll revert this.